core/include/esp/phy.h

```diff
@@ -0,0 +1,58 @@
+/** esp/phy.h
+ *
+ * PHY hardware management functions.
+ *
+ * These are not enough to configure the ESP8266 PHY by themselves
+ * (yet), but they provide utilities to modify the configuration set
+ * up via the SDK.
+ *
+ * Functions implemented here deal directly with the hardware, not the
+ * SDK software layers.
+ *
+ * Part of esp-open-rtos
+ * Copyright (C) 2016 ChefSteps, Inc
+ * BSD Licensed as described in the file LICENSE
+ */
+#ifndef _ESP_PHY_H
+#define _ESP_PHY_H
+
+#include <esp/types.h>
+#include <common_macros.h>
+
+#include <stdint.h>
+
+typedef enum {
+    BT_COEX_NONE,
+    BT_COEX_USE_BT_ACTIVE,
+    BT_COEX_USE_BT_ACTIVE_PRIORITY,
+} bt_coex_mode_t;
```

Comment:
"BT_COEX" smells like unnecessary contraction/abbreviation to me.  I can accept "BT" for "bluetooth" because it is a fairly standard abbreviation, but "coexist" is only three more characters than "coex" and is a lot more obvious what it means, particularly since that's what you've called it elsewhere already anyway (at the very least, it should be consistently named everywhere)..